modules/gcp/k8s/main.tf

```diff
@@ -12,7 +12,8 @@ resource "google_container_cluster" "tsb" {
   project            = var.project_id
   location           = var.region
   min_master_version = var.k8s_version
-
+  network            = var.vpc_id
+  subnetwork         = var.vpc_subnet
```

Comment:
Without this, a custom network was created, but the GKS cluster nodes were attached to the VPC `default` one.